srcalc/src/main/java/gov/va/med/srcalc/domain/Patient.java

```diff
@@ -160,6 +164,19 @@ public void setLabs(final Map<String, RetrievedValue> labs)
         fLabs = labs;
     }
     
+    /**
+     * Returns the patient's health factors in String form.
```

Comment:
Now that I understand the code, I think creating a tiny `HealthFactor` class would be much more self-documenting here.